image-builder/image_builder.sh

```diff
@@ -57,12 +57,30 @@ EOT
 exit "${error}"
 }
 
-while getopts "ho:s:" opt
+# Maximum allowed size in MB for root disk
+MAX_IMG_SIZE_MB=2048
+
+FS_TYPE=${FS_TYPE:-"ext4"}
+
+# Maximum no of attempts to create a root disk before giving up
+MAX_ATTEMPTS=5
+
+ATTEMPT_NUM=0
+
+while getopts "ho:s:f:" opt
 do
 	case "$opt" in
 		h)	usage ;;
 		o)	IMAGE="${OPTARG}" ;;
-		s)	IMG_SIZE="${OPTARG}" ;;
+		s)	IMG_SIZE=${OPTARG}
```

Comment:
If you run `./image_builder.sh -h`, there is no default value showing now as `IMG_SIZE` isn't set for me.

Reply:
Right, I am thinking of dropping `default` from output altogether, 

```
./image-builder/image_builder.sh -h
Usage: image_builder.sh [options] <rootfs-dir>
	This script will create a Kata Containers image file based on the 
	<rootfs-dir> directory.

Options:
	-h Show this help
	-o path to generate image file ENV: IMAGE
	-s Image size in MB ENV: IMG_SIZE

Extra environment variables:
	AGENT_BIN:  use it to change the expected agent binary name"
	USE_DOCKER: If set will build image in a Docker Container (requries docker)
	            DEFAULT: not set
```

What do you think? @jodh-intel

Reply:
Agreed. Could you also update that usage text to state that that size is auto-calculated so that `-s` (if specified) is just a "hint" really.

Reply:
Yes, that makes sense. Thanks.